app/assets/stylesheets/application.sass.scss

```diff
@@ -52,3 +52,20 @@ dialog::backdrop {
 .icon-primary-blue {
   filter: brightness(0) saturate(100%) invert(19%) sepia(28%) saturate(2849%) hue-rotate(197deg) brightness(94%) contrast(87%);
 }
+
+.unassign-recused-button {
+  color: #b50909;
+  border-radius: 0.15rem;
+  box-shadow: 0 0 0 2px #b50909;
+}
+
+.unassigned-row-bg {
+  @media screen and (max-width: 480px) {
+    td:first-child {
+      background-color: #f0f0f0 !important;
+    }
+  }
```

Comment:
I added the different shaded mobile header along with the unassigned bg color, and I also realized the tables were incorrect in the theme-custom-styles. Took me a while to figure out how to make it work together, but it looks like this works to me.